server/api/router.go

```diff
@@ -89,6 +89,7 @@ func createRouter(ctx context.Context, prefix string, svr *server.Server) *mux.R
 	apiRouter.HandleFunc("/config/cluster-version", confHandler.SetClusterVersion).Methods("POST")
 	apiRouter.HandleFunc("/config/replication-mode", confHandler.GetReplicationMode).Methods("GET")
 	apiRouter.HandleFunc("/config/replication-mode", confHandler.SetReplicationMode).Methods("POST")
+	apiRouter.HandleFunc("/config/ttl", confHandler.SetTTLConfig).Methods("POST")
```

Comment:
why not consist with store api,  use `/config?ttlSecond=xx?`

Reply:
updated.